.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: scala
+jdk: openjdk8
+
+before_install:
+  - sudo apt-get -y install nodejs
+
+cache:
+  directories:
+    - $HOME/.sbt
+    - $HOME/.ivy2/cache
+
+before_cache:
+  # Cleanup the cached directories to avoid unnecessary cache updates
+  - rm -fv $HOME/.ivy2/.sbt.ivy.lock
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
+  - find $HOME/.sbt        -name "*.lock"               -print -delete
+
+script:
+  - sbt scalafmtCheckAll
+  - sbt clean coverage cross/test
```

Comment:
@pishen `cross/test` this is due to the project name - `cross`. I'm thinking of rename to `minitimeCross`. 🤔